data.sql

```diff
@@ -14,4 +14,180 @@ VALUES
+How many different animals did Stephanie Mendez see?
+List all vets and their specialties, including vets with no specialties.
+List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+What animal has the most visits to vets?
+Who was Maisy Smith's first visit?
+Details for most recent visit: animal information, vet information, and date of visit.
+How many visits were with a vet that did not specialize in that animal's species?
+What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+ChatGPT
+To answer these questions, we can use the following queries:
+
+Who was the last animal seen by William Tatcher?
+vbnet
+Copy code
+SELECT animals.name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC
+LIMIT 1;
+This query joins the animals, visits, and vets tables, and filters by visits made by William Tatcher. It orders the results by the date of visit in descending order, so that the most recent visit comes first, and limits the output to the first row, which is the most recent visit. Finally, it selects the name of the animal seen during that visit.
+
+How many different animals did Stephanie Mendez see?
+sql
+Copy code
+SELECT COUNT(DISTINCT visits.animal_id)
+FROM visits
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez';
+This query joins the visits and vets tables, filters by visits made by Stephanie Mendez, and uses the COUNT DISTINCT function to count the number of unique animal IDs in the resulting set.
+
+List all vets and their specialties, including vets with no specialties.
+vbnet
+Copy code
+SELECT vets.name, specialties.name
+FROM vets
+LEFT JOIN specializations ON vets.id = specializations.vet_id
+LEFT JOIN species ON specializations.species_id = species.id
+ORDER BY vets.name, specialties.name;
+This query left joins the vets table with the specializations and species tables to include vets with no specialties. It selects the name of each vet and their corresponding specialty name, if applicable, and orders the results alphabetically by vet name and then by specialty name.
+
+List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+sql
+Copy code
+SELECT animals.name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez' AND visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+This query joins the animals, visits, and vets tables, and filters by visits made by Stephanie Mendez between April 1st and August 30th, 2020. It selects the name of the animal seen during each visit.
+
+What animal has the most visits to vets?
+vbnet
+Copy code
+SELECT animals.name, COUNT(visits.id) AS num_visits
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+GROUP BY animals.id
+ORDER BY num_visits DESC
+LIMIT 1;
```

Comment:
- [ Required ] I think it would be better if you remove any queries which are not related to inserting `data` from the `data.sql` file. Remember according to the project requirements all queries ought to be in the `queries.sql` file. Also, note that `.sql` files should not have plain text in them. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-05-05 11-42-41](https://user-images.githubusercontent.com/16392046/236414201-bf96632c-a10f-448e-a117-9512bc7e223e.png)